src/Extension/Tasks/DeprecationsCheckTask.yaml

```diff
@@ -0,0 +1,25 @@
+---
+id: "validation:php:deprecations-check"
+short_description: "Check your code by running a static code analysis and generate report with deprecations"
+help: ~
+stage: build
+version: 1.0.0
+command: "php %sdk_dir%/vendor/bin/psalm --config=%sdk_dir%/psalm.xml --no-progress --output-format=json --report=%reportDir%/psalm.deprecations_report.json 1> /dev/null"
```

Comment:
probably we need a configurable path here %sdk_dir%/psalm.xml. 
can you use file_exist(%project_dir%/psalm.xml.) ? %project_dir%/psalm.xml : %sdk_dir%/psalm.xml.
all rules will be in project

Reply:
added resolver